### PR TITLE
Add test plan for k4e

### PR DESCRIPTION
**Pull request by masayag**
Signed-off-by: Moti Asayag <masayag@redhat.com>

### CODE REVIEW

**Review by pkliczewski — APPROVED** [review]
I briefly went over the changes and they look good to me

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+The purpose of this project is to test the scalability and performance of [k4e project](https://github.com/jakub-dzon/k4e-operator) using [JMeter](https://jmeter.apache.org).
+
+## Installation
+The test plan for testing k4e is using the [Parallel Sampling controller](https://github.com/Blazemeter/jmeter-bzm-plugins/blob/master/parallel/Parallel.md) that needs to be installed.
+Instructions for installing plugins can be found on [here](https://jmeter-plugins.org/install/Install/).
+
+## Running the test plan
+Before the test can be run, the test plan needs to be prepared.
+The following parameters need to be set:
+* BEARER_TOKEN - the token is used to access the K8s API (see details below)
```
Comment (rgolangh) [inline README.md:11]: then maybe `K8S_BEARER_TOKEN` or something which doesn't have beaker in it?

Comment (masayag) [reply README.md:11]: well, there is no beaker mentioned anywhere in this PR ;-)
but your suggestion for the naming is better. I'll change it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+* HTTP_SERVER - K4E Server address
+* HTTP_SERVER_PORT - K4E Server port (default: 8888)
+* OCP_API_SERVER - K8S API server address
+* OCP_API_SERVER_PORT - K8S API server port (default: 8443)
+* NAMESPACE - K4E namespaces aren't supported yet, so the test will be run in the default namespace
+
+After the test plan is prepared, the test can be run:
+```bash
+JVM_ARGS="-Xms1g -Xmx4g -XX:MaxMetaspaceSize=512m" $JMETER_HOME/bin/jmeter.sh -n -t ./k4e-scale/jmeter/k4e_test_plan.jmx -l results.csv -e
```
Comment (bardielle) [inline README.md:20]: There isn't jmeter directory in that  repository 
I think you ment to wrote 
`JVM_ARGS="-Xms1g -Xmx4g -XX:MaxMetaspaceSize=512m" $JMETER_HOME/bin/jmeter.sh -n -t ./k4e-scale/test_plans/k4e_test_plan.jmx -l results.csv -e
`

File: README.md (hunk 1 above)
Comment (rgolangh) [reply README.md:11]: my bad, I read beaker. too many beakers lately.

File: README.md (hunk 2 above)
Comment (masayag) [reply README.md]: right, the patch should be `./test_plans/k4e_test_plan.jmx`, updated the PR.

**Pull request merged**